react/flags_omar/src/CountryAnswer.js

```diff
@@ -3,28 +3,36 @@ import './CountryAnswer.css';
 
 //Display 'Correct' or 'Incorrect'
 function CountryAnswer(props) {
-  const {wasCorrect, correctName} = props;
+
+  const {wasCorrect, correctName, optionsFunc} = props;
   // const correctName;
-  let answer;
+  let answer; 
   console.log(wasCorrect)
   if (wasCorrect === null) {
-    answer = (
+    answer=(
       <div className="answer-div">
         <br></br>
       </div>
     );
-  } else if (wasCorrect === false) {
+  } else {
     answer = (
       <div className="answer-div">
-        <div className="answer-text">Wrong. Correct answer: <strong>{correctName}</strong></div>
-        <button className="next">NEXT</button>
-      </div>
-    )
-  } else if (wasCorrect === true) {
-    answer = (
-      <div className="answer-div">
-        <div className="answer-text">Correct! <strong>{correctName}</strong></div>
-        <button className="next">NEXT</button>
+        <div className="answer-text">
+          {wasCorrect ? 
+            <span><span className="text-success">Correct! </span><strong>{correctName}</strong></span> : 
+
+            <span><span className="text-danger">Wrong. </span>Correct answer: <strong>{correctName}</strong></span>        
+          }
```

Comment:
Great job man,
small pointer for making the code little more compliant with eslint,
```
{
  wasCorrect 
  ? <span><span className="text-success">Correct! </span><strong>{correctName}</strong></span>
  : <span><span className="text-danger">Wrong. </span>Correct answer: <strong>{correctName}</strong></span>        
}
```
👍

Reply:
Hi there. I think I may be confused, I am not trying to edit the source code. I was just trying to download the files so I could practice on them. I thought I was committing to only my own repository, but now I'm seeing that I'm apparently trying to merge into the class' master. 

Please disregard my edits! I will discontinue pushing commits to the master repository .----.